src/utils/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const api = axios.create({
+    baseURL: "https://ac-nc-news-server.herokuapp.com/api"
+})
+
+export function getAllArticles() {
+    let path = '/articles'
```

Comment:
Is there a reason why you are putting the path into a variable instead of straight into the api.get()? If you are putting your path into a variable, please use a const if you are not reassigning.